freetextresponse/freetextresponse.py

```diff
@@ -178,10 +197,11 @@ def student_view(self, context=None):
             used_attempts_feedback=self._get_used_attempts_feedback(),
             submit_class=self._get_submit_class(),
             indicator_visibility_class=self._get_indicator_visiblity_class(),
-            word_count_message=self._get_word_count_message(
-                self.count_attempts
-            ),
             submitted_message=self._get_submitted_message(),
+            save_class=self._get_submit_class(),
```

Comment:
The original functionality, which may not exist in capa problems, is to hide the submit button with a 'nodisplay....something' tag if the user reached max attempts.  In any case, I figured if no submission then no saving sounded right.